### ISSUE TITLE
Ensure that README contains guidelines for the review process

### ISSUE THREAD

**Issue by emiltj** [emiltj/DANSK-gold-NER#17]
This should contain:
- Information that the wides span overrules a smaller span (e.g. 'Taler 8' should be tagged with PER, and not 8 with ORDINAL)
- Take off-set in the predictions by Roberta Large on ontonotes https://huggingface.co/tner/roberta-large-ontonotes5?text=High+court+ruled+him+guilty or/and on Internal Dialogue (write rebekah and Kenneth on Slack)
- Information on what we do in edge cases (write down docs of where there was doubt, and how it was resolved).  
- In short docs with multiple meanings (e.g. Doc = 'Pande', one might tag Pande as Verb or as Noun), Discard them (no parking sign)

**Comment by emiltj** [comment]
Also include GPE (since it was forgotten)
